pyproject.toml

```diff
@@ -1,64 +1,65 @@
-classifiers = ["Development Status :: 4 - Beta",
-               "Operating System :: Microsoft :: Windows",
-               "Operating System :: POSIX :: Linux",
-               "Operating System :: MacOS",
-               "Programming Language :: Cython",
-               "Programming Language :: Python :: 3 :: Only",
-               "Topic :: Office/Business :: Financial",
-               "License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)",
-               "Natural Language :: English"]
+name = 'dxfeed'
+version = '0.4.0'
+description = 'DXFeed Python API via C API'
+authors = ['Index Management Team <im@dxfeed.com>']
+build = 'build.py'
+license = 'MPL-2.0'
+readme = 'README.md'
+classifiers = ['Development Status :: 4 - Beta',
+               'Operating System :: Microsoft :: Windows',
+               'Operating System :: POSIX :: Linux',
+               'Operating System :: MacOS',
+               'Programming Language :: Cython',
+               'Programming Language :: Python :: 3 :: Only',
+               'Topic :: Office/Business :: Financial',
+               'License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)',
+               'Natural Language :: English']
 
-include = ["LICENSE", "dxfeed/**/*.c"]
-exclude = ["dxfeed/dxfeed-c-api/wrappers*",
-           "dxfeed/dxfeed-c-api/tests*",
-           "dxfeed/dxfeed-c-api/scripts*",
-           "dxfeed/dxfeed-c-api/lib*",
-           "dxfeed/dxfeed-c-api/docs*",
-           "dxfeed/dxfeed-c-api/.github*"]
+include = ['LICENSE', 'dxfeed/**/*.c']
+exclude = ['dxfeed/dxfeed-c-api/wrappers*',
+           'dxfeed/dxfeed-c-api/tests*',
+           'dxfeed/dxfeed-c-api/scripts*',
+           'dxfeed/dxfeed-c-api/lib*',
+           'dxfeed/dxfeed-c-api/docs*',
+           'dxfeed/dxfeed-c-api/.github*']
 
 [tool.poetry.urls]
-"Source code" = "https://github.com/dxFeed/dxfeed-python-api"
-"Documentation" = "https://dxfeed.readthedocs.io/en/latest/"
+'Source code' = 'https://github.com/dxFeed/dxfeed-python-api'
+'Documentation' = 'https://dxfeed.readthedocs.io/en/latest/'
 
 [tool.poetry.dependencies]
-python = "^3.6"
-pandas = "^0.25.1"
+python = '^3.6'
+pandas = '^0.25.1'
 # The following dependencies are used for docs generation when installed as extras
 # (e.g. pip install nornir[docs])
 # Currently they have to be specified as non-dev dependencies with optional = true
 # Refer to: https://github.com/sdispater/poetry/issues/129
 # Issue to watch for: https://github.com/python-poetry/poetry/issues/1644
-toml = { version = "^0.10.0", optional = true }
-jupyter = { version = "^1.0.0", optional = true }
+toml = { version = '^0.10.0', optional = true }
+jupyter = { version = '^1.0.0', optional = true }
 
 [tool.poetry.dev-dependencies]
-cython = "^0.29.13"
-taskipy = "^1.1.3"
-pytest = "^5.3.5"
-sphinx = "^2.4.4"
-toml = "^0.10.0"
-sphinx_rtd_theme = "^0.4.3"
-pygments = "^2.6.1"
+cython = '^0.29.13'
+taskipy = '^1.1.3'
+pytest = '^5.3.5'
+sphinx = '^2.4.4'
+toml = '^0.10.0'
```

Comment:
optional dependencies not the same thing as dev-dependencies. Toml may be included in user deps in order to implement __version__, in that case, it will be removed from dev-deps